sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -310,4 +311,17 @@ private[sql] abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
       case _ => Nil
     }
   }
+
+  object DDLStrategy extends Strategy {
```

Comment:
Actually, I am not sure we should put them in BasicOperators. We cannot just create a `RunnableCommand` in `ddl.scala` since `SQLContext` does not allow persistent table and we need to throw the error in `SparkStrategies`. Also, I feel code is clear when we put stuff related to the data source API together.